static/js/jira-connect-ghe.js

```diff
@@ -0,0 +1,64 @@
+const ALLOWED_PROTOCOLS = ["http:", "https:"];
+const GITHUB_CLOUD = ["github.com", "www.github.com"];
+
+/**
+ * Method that checks the validity of the passed URL
+ *
+ * @param {string} inputURL
+ * @returns {boolean}
+ */
+const checkValidUrl = inputURL => {
+	try {
+		const { protocol, hostname } = new URL(inputURL);
+
+		if (!ALLOWED_PROTOCOLS.includes(protocol)) {
+			return false;
+		}
+		if (hostname.split('.').length < 2) {
```

Comment:
.... this is to check .com .org .net etc??
Why do we need to check this?

what you originally had was clearer with intent, but if this catches everything then this could be fine, 
is localhost or server names valid hosts?

maybe just a comment above the if statement what its trying to achieve?

Reply:
As mentioned by @rachellerathbone , the reason for this change is to ensure that there is an extension at the end like `.com` or `.net`, but we don't want to do any specific checks like before